nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
@wschenck I think I may be confused about the code we're discussing. Could you add the snippet or a link to it? Currently 791 on the master is to:
```
    } while ( to_do_ > 0 and not exit_on_user_signal_
      and not exceptions_raised.at( thrd ) );

>>>>>    // End of the slice, we update the number of synaptic elements
    for ( std::vector< Node* >::const_iterator i =
            kernel().node_manager.get_nodes_on_thread( thrd ).begin();
          i != kernel().node_manager.get_nodes_on_thread( thrd ).end();
          ++i )
    {
      ( *i )->update_synaptic_elements(
        Time( Time::step( clock_.get_steps() + to_step_ ) ).get_ms() );
    }

  } // end of #pragma parallel omp
```
which is in the same place in the  current *monster* update_ () function, right after the update loop. That needs to be refactored! 200 lines seperating the do {} while... and the function still has another 100 lines...